core/genesis.go

```diff
@@ -374,7 +374,7 @@ func DefaultAlfajoresGenesisBlock() *Genesis {
 }
 
 // DeveloperGenesisBlock returns the 'geth --dev' genesis block.
-func DeveloperGenesisBlock(period uint64, faucet common.Address) *Genesis {
+func DeveloperGenesisBlock() *Genesis {
```

Comment:
nit: I'd provide these parameters (like `15, common.Address{}`) in the genesis test rather than removing them from places that did use the overrides

Reply:
What do you mean by **overrides**? Those deleted params are not used

Reply:
Oh, I didn't see that they were not being used. In this case it makes sense to either use them or not pretend to be able to override them.